### PR TITLE
Added GUI for database

### LINKED ISSUE #1: Create basic GUI to use with database

### LINKED ISSUE #3: Create page for .csv file

### LINKED ISSUE #4: Create buttons for CRUD operations

### LINKED ISSUE #5: Create button/page to view reports

### LINKED ISSUE #10: Create implementation to view all documentation.

### LINKED ISSUE #11: Link buttons in GUI to correspond with methods/implementations

### PR DESCRIPTION
**Pull request by FindingDorri**
All components is functional except for the menu bar.

### CODE REVIEW

**Review by SHamil36 — APPROVED** [review]
This looks great and looks like you have implemented all the stuff that needs to be there. For the future I would just recommend more documentation for the code and we can look at adding test later on.

**Review by SHamil36 — CHANGES_REQUESTED** [review]
Can you rename the file to database.java and resubmit the pull request?

**Comment by SHamil36** [comment]
This  would make it so there is not two different files with the same code.

**Comment by FindingDorri** [comment]
I'll add more documentation and change the name.

**Review by SHamil36 — APPROVED** [review]
We  will commit this code and we can make changes with more documentation and implementation in the next commit.

File: Table.java [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+	
+	/** The item table */
+	private JTable table;
+	private DefaultTableModel tableModel;
+
+	/** Connection to database */
+	private Connection con;
+	
+	/**
+	 * Launch the application.
```
Comment (CameronSpear) [inline Table.java:55]: We're looking at adjusting the CRUD methods so that it calls TeamMySQL database, rather than putting it in here.

**Review by CameronSpear — APPROVED** [review]
See comment 50 or so lines in, other than that it looks pretty good. I like the test methods for empty fields, they look good and give a bit more testing.

**Pull request merged**